PublicMapping/districtbuilder
# Calculate Polsby Popper compactness for drawn districts

## Overview
Calculate Polsby Popper compactness for drawn districts

### Demo
![db_polsby_popper](https://user-images.githubusercontent.com/2926237/88820353-381a5600-d18f-11ea-924e-154d226b3b3c.gif)

### Notes

I've never used Turf before but it's recommended that you install each package separately, which I did. Each package comes with its own TS types.

Also fixed a small alignment bug with the label dropdown in Chrome

I added a little `assertNever` function that makes it easier to do exhaustive checking. In this case it's used to ensure we return something for each possible value that `CompactnessScore` could take on

## Testing Instructions
* Draw some districts and verify that the more compact ones generally show a higher percentage
* Draw a non-contiguous district and ensure that the 'X' icon is shown with the appropriate tooltip

Closes #118
